webpack/ForemanTasks/Components/common/ActionButtons/ActionButton.js

```diff
@@ -1,27 +1,38 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { CancelButton } from './CancelButton';
-import { ResumeButton } from './ResumeButton';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { ActionButtons } from 'foremanReact/components/common/ActionButtons/ActionButtons';
 
 export const ActionButton = ({ id, name, availableActions, taskActions }) => {
   const isResume = availableActions.resumable;
+  const isCancel = availableActions.cancellable;
+  const buttons = [];
+  let title;
+  if (!isResume && !isCancel) {
+    title = __('Task cannot be canceled');
+  }
   if (isResume) {
-    return (
-      <ResumeButton
-        id={id}
-        name={name}
-        onClick={taskActions.resumeTask}
-        disabled={false}
-      />
-    );
+    buttons.push({
+      title: __('Resume'),
+      action: {
+        disabled: !isResume,
+        onClick: () => taskActions.resumeTask(id, name),
+      },
+    });
+  }
+  if (isCancel || !isResume) {
+    buttons.push({
+      title: __('Cancel'),
+      action: {
+        disabled: !isCancel,
+        onClick: () => taskActions.cancelTask(id, name),
+      },
+    });
   }
   return (
-    <CancelButton
-      id={id}
-      name={name}
-      disabled={!availableActions.cancellable}
-      onClick={taskActions.cancelTask}
-    />
+    <div title={title}>
```

Comment:
should we use `OverlayTrigger` here instead?

Reply:
I don't think we should as it's here only to provide extra basic information

Reply:
ok, should we use a `span` instead?

Reply:
I'm not sure what you mean, instead of what?

Reply:
oh, I mean instead of the `div`, since it's an inline feature from what I understand,
if it's being rendered inside a table cell.

Reply:
Yeah that makes sense, thanks!